retroarch.cfg

```diff
@@ -138,6 +96,27 @@
 # By default, tries to use first suitable driver.
 # video_context_driver =
 
+# Audio driver backend. Depending on configuration possible candidates are: alsa, pulse, oss, jack, rsound, roar, openal, sdl, xaudio.
+# audio_driver =
+
+# Audio resampler driver backend. Which audio resampler to use.
+# Default will use "sinc".
+# audio_resampler =
+
+# Camera driver.
+# camera_driver =
+
+# Location driver.
+# location_driver =
+
+# Menu driver to use. "rgui", "lakka", etc.
```

Comment:
Also out of scope of this PR but you can take the opportunity to change from

`+#` Menu driver to use. "rgui", "lakka", etc.
to
`+#` Menu driver to use. "rgui", "xmb", "glui", etc.

Thank you.

Reply:
Should "etc" be removed if all three menu drivers are listed?

Reply:
There are other menu drivers which are not enabled by default.